cmd/arescli/main.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"github.com/abiosoft/ishell"
+	"github.com/spf13/cobra"
+	"net/http"
+	"fmt"
+	"github.com/fatih/color"
+	"io/ioutil"
+	"encoding/json"
+	"strings"
+)
+
+const contentType = "application/json"
+
+type shellContext struct {
+	host string
+	port int
+	clusterMode bool
+	client http.Client
+}
+
+func Execute() {
+	var ctx shellContext
+
+	// ishell shell
+	shell := ishell.New()
+
+	shell.Println("Welcome to AresDB Cli!")
+	shell.AddCmd(&ishell.Cmd{
+		Name: "show",
+		Help: "`show tables` will show all tables in current cluster",
+		Func: func(c *ishell.Context) {
+			hint := "did you mean `show tables`?"
```

Comment:
better put the function in real separate function instead of putting all functions in one huge function